TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/Accumulation.java

```diff
@@ -47,6 +47,9 @@ public String toString() {
         final StringBuilder sb = new StringBuilder("Accumulation{");
         sb.append("rrPair=").append(rrPair);
         sb.append(", state=").append(state);
+        if (trafficStreamsSorter.hasPending()) {
```

Comment:
Any reason to only output this with pending? Could it be useful to know the offset or items even if the accumulation is caught up. I could also see this being unnecessary noise so I think I'm fine for now until I find a case where I actually want this regardless

Reply:
It would just be empty, so when you see an Accumulation w/out this part, you can infer that trafficStreamsSorter has no elements within it.

Reply:
The offset count would still be shown, but I don't really see a need to have that right now